src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -129,6 +131,43 @@ public static int parseInsurancePlan(String insurancePlanId) throws ParseExcepti
         return trimmedInsurancePlanId;
     }
 
+    /**
+     * Parses {@code String claimId} into a valid claimId.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given claimId is invalid based on preset conventions.
+     */
+    public static String parseClaimId(String claimId) throws ParseException {
+        requireNonNull(claimId);
+        String trimmedClaimId = claimId.trim();
+        Claim.checkValidClaimId(trimmedClaimId);
+        return trimmedClaimId;
+    }
+
+    /**
+     * Parses {@code String claimId} into a valid claimId.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the claim amount is not a valid positive integer
+     */
+    public static int parseClaimAmount(String claimAmount) throws ParseException {
+        requireNonNull(claimAmount);
+        int claimAmountInt;
+        try {
+            int centsInADollar = 100;
+            int claimAmountDollars = Integer.parseInt(claimAmount.trim().split("\\.")[0]);
+            if (claimAmount.trim().split("\\.")[1].length() > 2) {
```

Comment:
Can we just use claimAmountCents as calculated below already? I think its abit confusing to the reader what `claimAmount.trim().split("\\.")[1].length()` means. In the future, we could abstract it out as a method to avoid magic literals like 2, but this is just a nitpick. E.g. `if (Claim.isNotValidCents(claimAmountCents)) {}` but I'm not sure which is the best class to handle it.

Reply:
Yep, sure that improves readability, will fix!